gui/src/types.ts

```diff
@@ -4,6 +4,8 @@ export const generalSettingsSchema = z.object({
   darkMode: z.enum(["auto", "dark", "light"]),
   abiWatch: z.boolean().default(false),
   abiWatchPath: z.string().optional(),
+  hideEmptyTokens: z.boolean().default(true),
```

Comment:
this shouldn't need a default on the frontend. that comes from the backend

Reply:
Yeah, noticed that as well. The reason i put a default was because the other fields have one as well. Look at abiWatch for instance

Reply:
yeah, that one is wrong too. my bad 😅